### DISCUSSION TITLE
Question about state being saved in Macos

### DISCUSSION THREAD

**Discussion by juangentile** [ghostty-org/ghostty#3533] [Q&A] [answered]
Hello! I'm trying to understand how the saving of the state works in Macos. I have 2 different computers (both macos latest version) and in both I've installed (with brew) and configure ghostty in the same way. But in one of them the state is not saved unless I set window-save-state to always. In the documentation it says the following:
``default` will use the default system behavior. On macOS, this
will only save state if the application is forcibly terminated
or if it is configured systemwide via Settings.app.`
For the machine where the value is default and the state is saved, I don't remember manually setting anything. Does anyone know what Macos config that is?

Thank you,

**Comment by markpeek** [comment, marked as answer]
Try checking System Settings > Desktop & Dock > Windows > Close windows when quitting an application

**Reply by juangentile** [reply]
Thank you! That was it!
